### PR TITLE
GTT-932 Frontend to display dashboard audit logs

### PR DESCRIPTION
**Pull request by ferdingler**
## Description

- Added new screen to display dashboard audit logs. 

## Testing

You can test in my personal environment: 
https://fdingler.badger.wwps.aws.dev/admin/dashboard/6f6f93e6-3f56-455e-8a06-60cfa3cb0080/history 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by miguelpdiaz8** [comment]
I'm getting a 403 when loading the page in your environment for the settings endpoint

**Comment by ferdingler** [comment]
Sorry I forgot to assign you a role. Can you try again please? @miguelpdiaz8

File: frontend/src/containers/DashboardHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    ],
+    []
+  );
+
+  const onSearch = (query: string) => {
+    setFilter(query);
+  };
+
+  if (!dashboard) {
+    return <Spinner label="Loading..." />;
```
Comment (miguelpdiaz8) [inline frontend/src/containers/DashboardHistory.tsx]: The Loading icon is appearing on the left of the page, can you center it with something like:

<Spinner className="text-center margin-top-9" label="Loading" />

**Review by miguelpdiaz8 — CHANGES_REQUESTED** [review]

File: frontend/src/containers/DashboardHistory.tsx (hunk 1 above)
Comment (ferdingler) [reply frontend/src/containers/DashboardHistory.tsx]: Good idea.

**Review by miguelpdiaz8 — APPROVED** [review]

**Pull request merged**